README.md

```diff
@@ -262,6 +262,29 @@ cargo {
 }
 ```
 
+### exec
+
+This is a callback taking the `ExecSpec` we're going to use to invoke cargo, and
```

Comment:
s/we're going to use to invoke `cargo`/that will be used to invoke `cargo build`/.